compiler/src/codegen/transl_anf.re

```diff
@@ -1303,6 +1351,55 @@ let transl_signature = (~functions, ~imports, signature) => {
             )
           };
         }
+      | TSigType(tid, {type_kind: TDataVariant(cds)} as td, rs) => {
```

Comment:
Just so I understand, can you clarify what this is doing? (In GitHub is fine)

Reply:
Absolutely! It just exports the functions that were created for the types you defined in the module. It won't be necessary anymore when we're not creating functions to create variants.